.agents/skills/agents-md/SKILL.md

```
@@ -48,6 +48,16 @@ readonly MAX_RETRIES=3
 - [ ] Security vulnerabilities
 ```
 
+### 4. Development Workflow Rules
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)